install.sh

```diff
@@ -2148,6 +1580,39 @@ DATASTORE_DRIVER_ISCSI_SCRIPTS="src/datastore_mad/remotes/iscsi_libvirt/cp \
                          src/datastore_mad/remotes/iscsi_libvirt/snap_flatten \
                          src/datastore_mad/remotes/iscsi_libvirt/clone"
 
+DATASTORE_DRIVER_RSYNC_SCRIPTS="src/datastore_mad/remotes/rsync/cp \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

DATASTORE_DRIVER_RSYNC_SCRIPTS appears unused. Verify use (or export if used externally).